test/test-tcp-open.c

```diff
@@ -158,9 +201,34 @@ static void connect_cb(uv_connect_t* req, int status) {
 }
 
 
+static void connect1_cb(uv_connect_t* req, int status) {
+  uv_buf_t buf = uv_buf_init("P", 1);
+  uv_stream_t* stream;
+  int r;
+
+  ASSERT(req == &connect_req);
+  ASSERT(status == 0);
+
+  stream = req->handle;
+  connect_cb_called++;
+
+  r = uv_timer_init(uv_default_loop(), &tm);
+  ASSERT(r == 0);
+
+  r = uv_timer_start(&tm, timer_cb, 2000, 0);
+  ASSERT(r == 0);
+
+  r = uv_write(&write_req, stream, &buf, 1, write_cb);
+  ASSERT(r == 0);
+
+  /* Start reading */
+  r = uv_read_start(stream, alloc_cb, read_cb);
+  ASSERT(r == 0);
+}
```

Comment:
Nothing seems to use these new functions?

Reply:
Sorry, fixed.